### ISSUE TITLE
MiniProfiler storage failure: can't modify frozen

### ISSUE THREAD

**Issue by wage-coolie** [MiniProfiler/rack-mini-profiler#611]
MiniProfiler storage failure: can't modify frozen String: "<!DOCTYPE html>\n<html>\n  <head>\n    <title>Rack::MiniProfiler Requests</title>\n  </head>\n  <body>\n  </body>\n</html>\n"

Rails 7

Rack::MiniProfiler.config.auto_inject = true
Rack::MiniProfiler.config.show_total_sql_count = true
Rack::MiniProfiler.config.enable_advanced_debugging_tools = true
Rack::MiniProfiler.config.enable_hotwire_turbo_drive_support = true
Rack::MiniProfiler.config.profile_parameter = true
These are my settings, 
 Upon hitting the /requests urrl i am getting this error in console.Also turbo streams seems to be not loading new data in speed badge, as a matter of fact my speed badge is dissappearing after every turbo stream request. If anyone have solution to these issues. Pls respond

**Comment by krzkrzkrz** [comment]
This is what I am seeing as well. @wage-coolie did you manage to figure this out?

**Comment by wage-coolie** [comment]
> This is what I am seeing as well. @wage-coolie did you manage to figure this out?

Nope, I am not able to work it out at all with rails rails7, it sesms turbo requests reload doesn not load a new mini-profiler log, i am now forced to use rails panel extensions.

**Comment by teoulas** [comment]
It seems this is caused by the frozen string here: https://github.com/MiniProfiler/rack-mini-profiler/blob/master/lib/mini_profiler/views.rb#L1

The [blank_page_html](https://github.com/MiniProfiler/rack-mini-profiler/blob/master/lib/mini_profiler/views.rb#L93-L105) method returns this frozen string, which I'm guessing miniprofiler tries to modify, to inject the script which shows the timings widget.

I've tried removing the frozen string literal magic comment from the gem code locally and it works.

**Comment by pedrohcarvalhom** [comment]
Any solution? I'm still facing this problem
